File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.java
class Solution {
    public int ans(int a){
    int i = 1;
    int tmp = 0;
    while(i <= a){
        if(a % i == 0){
            tmp++;
        }
       i++;
    }
    
    if(tmp > 2) return 1;
    else return 0;
}
    public int solution(int n) {
        int answer = 0;
          
    for(int i=4;i<=n;i++){
        answer += ans(i);
    }
        return answer;
    }
}

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.py
def solution(n):
    answer = 0
    
    for i in range(4, n+1):
        tmp = 1
        for j in range(2, i+1):
            tmp += i % j == 0 and 1 or 0
    
        if tmp > 2 :
            answer += 1
    
    return answer